src/styles/blocks/cards.scss

```
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  width: $cardWidth;
+
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    @extend %font-500;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    @extend %font-400;
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__mark {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 24px;
+  }
+
+  &__comment {
+    @extend %font-400;
+    align-self: center;
+    color: #060b35;
+  }
+
+  &__price-wrapper {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    @extend %font-400;
+
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-teg {
+    @extend %font-700;
+  }
+
+  &__pay-btn {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+    text-decoration: none;
+
+    width: 166px;
+    height: 40px;
+
+    border-radius: 5px;
+    background: #00acdc;
+
+    &:hover {
+      background: #fff;
+      color: #00acdc;
+      border: 1px solid #00acdc;
```

Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
```suggestion
      border-color: #00acdc;
```